### PR TITLE
Suppliers fixes

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: src/Service.Host/client/src/components/RunMrp.js
Comment (richardiphillips) [inline src/Service.Host/client/src/components/RunMrp.js:5]: don't do this to me!

File: src/Service.Host/client/src/components/RunMrp.js
Comment (richardiphillips) [inline src/Service.Host/client/src/components/RunMrp.js:25]: this is changed on my branch. don't merge hell me

**Review by richardiphillips — APPROVED** [review]

File: src/Service.Host/client/src/components/RunMrp.js
Comment (lewisrenfrew) [reply src/Service.Host/client/src/components/RunMrp.js:25]: I just commented it out to make the linter be quiet. I'll revert before I merge

**Pull request merged**
